openprescribing/measure_definitions/statinintensity.json

```diff
@@ -28,81 +28,59 @@
   "is_percentage": true,
   "is_cost_based": false,
   "low_is_good": true,
-  "numerator_type": "bnf_items",
-  "numerator_bnf_codes_query": [
-    "SELECT bnf_code FROM {hscic}.presentation WHERE",
-    "name LIKE '%tablet%' OR",
-    "name LIKE '% tab %' OR",
-    "name LIKE '% tab' OR",
-    "name LIKE '% tabs %' OR",
-    "name LIKE '% tabs' OR",
-    "name LIKE '%capsule%' OR",
-    "name LIKE '% caps %' OR",
-    "name LIKE '% caps' OR",
-    "name LIKE '%caplet%' OR",
-    "name LIKE '%Chewtab%' OR",
-    "name LIKE '%chewable tablet%'"
+  "numerator_type": "custom",
+  "numerator_columns": [
+    "SUM(p.items) AS numerator"
   ],
-  "numerator_bnf_codes_filter": [
-    "0212000AA # Rosuvastatin Calcium",
-    "0212000AC # Simvastatin & Ezetimibe",
-    "0212000B0 # Atorvastatin",
-    "0212000C0 # Cerivastatin",
-    "0212000M0 # Fluvastatin Sodium",
-    "0212000X0 # Pravastatin Sodium",
-    "0212000Y0 # Simvastatin",
-    "~0212000AA%AA # Rosuvastatin Calc_Tab 10mg (brand, generic)",
-    "~0212000AA%AG # Rosuvastatin 10mg capsules (brand, generic)",
-    "~0212000AA%AB # Rosuvastatin Calc_Tab 20mg (brand, generic)",
-    "~0212000AA%AH # Rosuvastatin 20mg capsules (brand, generic)",
-    "~0212000AA%AC # Rosuvastatin Calc_Tab 40mg (brand, generic)",
-    "~0212000AA%AI # Rosuvastatin 40mg capsules (brand, generic)",
-    "~0212000B0%AB # Atorvastatin_Tab 20mg (brand, generic)",
-    "~0212000B0%AM # Atorvastatin 20mg chewable tablets sugar free (brand, generic)",
-    "~0212000B0%AN # Atorvastatin_Tab 30mg (brand, generic)",
-    "~0212000B0%AC # Atorvastatin_Tab 40mg (brand, generic)",
-    "~0212000B0%AP # Atorvastatin_Tab 60mg (brand, generic)",
-    "~0212000B0%AD # Atorvastatin_Tab 80mg (brand, generic)",
-    "~0212000Y0%AH # Simvastatin_Tab 80mg (brand, generic)",
-    "~0212000AC%AC # Simvastatin/Ezetimibe_Tab 80mg/10mg (brand, generic)"
+ "numerator_from": [
+  "{hscic}.normalised_prescribing p ",
+  "INNER JOIN {dmd}.vmp vmp ON CONCAT(SUBSTR(p.bnf_code,0,9),'AA', SUBSTR(p.bnf_code,-2), SUBSTR(p.bnf_code,-2)) = vmp.bnf_code -- joins prescribing data to vmp table using generic BNF code \n",
```

Comment:
because it's creating a full generic BNF code, which always replicates the last two digits, as described:
![image](https://user-images.githubusercontent.com/18683239/171162261-6cdcc4e3-d818-42e4-b918-a4e181bc3fff.png)

Reply:
Oh yes, I'd forgotten that